### DISCUSSION TITLE
Project paused failed to restore

### DISCUSSION THREAD

**Discussion by mauriciopettinato** [supabase/supabase#32395] [Questions]
I tried to restore a paused project, but it failed. The data is theoretically intact, and I can download the backup. I opened a ticket 2 weeks ago, but customer support has been quite unresponsive and no solution so far to restore it. Does anyone have any idea on how to get my backup file and restore into a new database?

**Comment by GaryAustin1** [comment]
Was this past 90 days when you tried to restore or is it hung in the restore process?   You might ping them again as they usually solve the hung restore issue.

This is an updated guide they generated recently to restoring from backup... https://supabase.com/docs/guides/platform/upgrading#pause-and-restore

**Reply by Dramelac** [reply]
I have the same problem 6 days after pausing:
![image](https://github.com/user-attachments/assets/ebd986d2-dafb-491d-afa4-158f9af4f823)

**Reply by adeelchainz** [reply]
Having the same issue, not while backing up but randomly when testing the app, integrated with flutter flow

**Comment by adeelchainz** [comment]
Having the same issue while testing my app integrated with flutter flow, not restoring the backup
